### DISCUSSION TITLE
Lunch & Learn: Open Source Project Walkthrough, September 3 2021

### DISCUSSION THREAD

**Discussion by danieltott** [Virtual-Coffee/virtualcoffee.io#303] [Events]
**What:** [Open Source Project Walkthrough](https://meetingplace.io/virtual-coffee/events/6906)
**When:** September 3, 2021, 1pm EDT
**Who:** Dan Ott

Let's tour an Open Source Project on GitHub! We'll walk through pretty much every piece of an open source repository, and go behind the scenes and see what it's like as a maintainer as well.

Leading up to the event, I would love to hear suggestions about what you'd like to see, or answer any questions about the event in general! I have my own list running, but if there's anything in particular you've been wondering about when it comes to Open Source or GitHub, drop them here in the comments and I'll try to address them on Friday!! 

[RSVP on Meetingplace](https://meetingplace.io/virtual-coffee/events/6906) - the Zoom link will be posted in Slack the day of. See you Friday!

**Comment by maelingmurphy** [comment]
Looking forward to this!

As a potential open source contributor, what steps should you take when you first identify an issue you may be interested in working on? Or when you have a feature you are interested in proposing? Looking for advice on best practices with these two approaches.

**Comment by Cerchie** [comment]
From a maintainer's perspective-- what makes a `good first issue`? I've labeled some of mine that way but some of them require (just a little) pre-req knowledge. If I document what needs to be learned can these still be good first issues?

**Reply by BekahHW** [reply]
@Cerchie maybe drop this in the open-source slack channel for some feedback if you're still struggling.

**Comment by danieltott** [comment]
Thanks everyone for coming! Here's some links from the presentation/chat: 

### Tech equipment (pre-talk chatting)

- https://www.iamdeveloper.com/pages/uses/
- https://www.logitech.com/en-ca/products/webcams/brio-4k-hdr-webcam.html
- https://shift.infinite.red/the-remote-work-starter-kit-desk-stuff-edition-6b841b8c745

### GitHub Resources

- https://github.com/nickytonline/nickytonline/blob/main/.github/workflows/blog-post-workflow.yml
- https://dev.to/jarvisscript/how-to-build-your-github-profile-page-23jb
- https://github.com/github/dev/
- https://guides.github.com/features/mastering-markdown/

### Things we looked at:

- https://github.com/Virtual-Coffee/virtualcoffee.io
- https://github.com/Virtual-Coffee/virtualcoffee.io/blob/main/CONTRIBUTING.md#creating-a-pull-request
- https://github.com/Virtual-Coffee/virtualcoffee.io/discussions/303

**Comment by danieltott** [comment]
Here's the recording! https://www.youtube.com/watch?v=WuIhnUjIFbM
